docs/Datadog.Trace.Annotations/README.md

```diff
@@ -0,0 +1,15 @@
+# Datadog.Trace.Annotations NuGet package
+
+This package contains custom attribute types to enable additional features of the Datadog APM instrumentation library.
+
+> Note: Automatic instrumentation is required for the attributes in this package to take effect. Please [read our documentation](https://docs.datadoghq.com/tracing/setup/dotnet) for details on how to install the tracer for automatic instrumentation.
+
+> Note: If you are unable to add new package references to your application, you may still enable this functionality by defining types inside your application whose full name and type members match the definitions in this package.
```

Comment:
Could we provide the minimal code for this? Maybe right here in the readme, or link to a `.cs` file in the repo.

Reply:
In the list of attributes below I provided a link to the repo, but I think we could also add the code snippet since this package is pretty small. I'll go ahead and add that

Reply:
Added in fe2336d80ad6a94278baf7da8ae5960381427772